R/split_moment_matching.R

```diff
@@ -0,0 +1,128 @@
+#' Split moment matching for efficient approximate leave-one-out cross-validation (LOO)
+#'
+#' A function that computes the split moment matching importance sampling loo.
+#' Takes in the moment matching total transformation, transforms only half
+#' of the draws, and computes a single elpd using multiple importance sampling.
+#'
+#'
+#' @export split_mmloo
```

Comment:
export statements do not need to be more specific if they appear in the doc of a function that should be exported via this statement.

Reply:
Just to be clear, you mean that we can simply put "@export" ?